src/main/java/com/aviva/APIBoundary/CarController.java

```diff
@@ -32,17 +32,21 @@ public String index() {
      * Returns a list of recommended cars for a certain client.
      * @param clientID the client's ID number
      * @return list of recommended cars, formatted as a JSON string
-     * @throws IOException
-     * @throws InterruptedException
-     * @throws ClassNotFoundException
+//     * @throws IOException
+//     * @throws InterruptedException
+//     * @throws ClassNotFoundException
      */
     // TODO: document the format of the returned JSON
     @PostMapping("/cars")
-    public String getCars(@RequestBody String clientID) throws IOException, InterruptedException, ClassNotFoundException {
-        AccountHolderDataInterface accountData = new CSVAccountHolderData();
-        AccountHolder user = accountData.getClientByID(clientID);
-        JSONObject cars = CarRecommender.getRecommendedCars(user);
-        System.out.println(cars);
+//    public String getCars(@RequestBody String clientID) throws IOException, InterruptedException, ClassNotFoundException {
+//        AccountHolderDataInterface accountData = new CSVAccountHolderData();
+//        AccountHolder user = accountData.getClientByID(clientID);
+//        JSONObject cars = CarRecommender.getRecommendedCars(user);
```

Comment:
Since there is some common code between the stuff that is commented out and the stuff that is not, I think it would create less clutter if instead of doing this, you just

add lines 42 and 43 inside the method as comments